File: Python/3307.py
from math import sqrt
ver = 0
azul = 0
ama = 0
for i in range(int(input())):
    n = int(input())
    r = sqrt(n/(4*3.14))
    if r < 12:
        print(f'vermelho = R$ {n*0.09:.2f}')
    elif r <= 15:
        print(f'azul = R$ {n*0.07:.2f}')
    else:
        print(f'amarelo = R$ {n*0.05:.2f}')

File: C/3307.c
#include<stdio.h>
#include <math.h>

int main() {
    int n; scanf("%d", &n);
    for(int i = 0; i < n; i++){
        int x; scanf("%d", &x);
        double r = sqrt(x/(4*3.14));
        if(r < 12)
            printf("vermelho = R$ %.2lf\n", x*0.09);
        else if(r <= 15)
            printf("azul = R$ %.2lf\n", x*0.07);
        else
            printf("amarelo = R$ %.2lf\n", x*0.05);
    }
    
}
